platform-core-module/src/main/java/com/softicar/platform/core/module/user/AGUser.java

```diff
@@ -69,10 +69,9 @@ public static AGUser getSystemUser() {
 		return AGCoreModuleInstance.getInstance().getSystemUser();
 	}
 
-	@Override
-	public Boolean isSuperUser() {
+	public Boolean isCoreModuleAdmin() {
```

Comment:
isAdministrator()?

Reply:
**TL;DR:**
I think we _must_ inline `isCoreModuleAdmin` from both `AGUser` and `SofticarAjaxSession`.

**Long version:**
I think this method should not exist here in the first place. The same goes for the equally-named method on `SofticarAjaxSession`. Those feel out of place, and they're like nudging towards a cliff.
Payload software will handle `AGUser` objects all over the place; so `isCoreModuleAdmin`/`isAdministrator` might be called because it appears (!) to be a convenient way to check for administrative permissions.
But it's not. Because it's the wrong way to go; beyond the core module, that is. No other module should ever check for `CorePermissions.ADMINISTRATION`, so we should not make that check readily-available (on `AGUser`).
Also, we got a terminological mess here. If anything, it should be `hasAdministration`, instead of `isAdministrator` - because there is no such thing as a hard-coded "administrator" role in the core module (anymore). That mess can be avoided altogether - see the TL;DR above.